Sources/hostmgr-helper/VirtualMachineSlot.swift

```diff
@@ -150,15 +66,15 @@ class VirtualMachineSlot: NSObject, ObservableObject {
             "Stopping \(clean ? "and cleaning " : "")\(role.displayName) slot with current status \(state)."
         )
         switch state {
-        case .starting(let config, let task):
-            // MARK: Starting -> Stopping
-            self.state.transitionTo(newState: .stopping(config))
+        case .starting(let mvm, let task):
+            try state.transitionTo(.stopping(mvm))
             task.cancel()
+            await mvm.stop()
```

Comment:
This was added for the event where the cancellation was [caught here](https://github.com/Automattic/hostmgr/pull/140/files#diff-ea4122eff37a787d0639f5d4fb208d830966113e853b89f260ffb2e58341fb46R37) - we'd need to handle it like a running VM at that point, I think.